Tribler/Test/Core/Libtorrent/test_libtorrent_download_impl.py

```diff
@@ -140,3 +140,60 @@ def test_multifile_torrent(self):
         test_dict["a"] = "b"
         pstate.set("state", "engineresumedata", test_dict)
         return impl.network_create_engine_wrapper(pstate)
+
+    @deferred(timeout=20)
+    def test_save_resume(self):
+        """
+        testing call resume data alert
+        """
+        tdef = TorrentDef()
```

Comment:
Can you please refactor this creation of a torrent into a separate method? You are duplicating this in `test_save_resume` and `test_save_resume_exists`.